src/main/resources/static/test-data.tsv

```diff
@@ -1,11 +1,11 @@
-EventName	CRFname	CRFversion	item1	item2	item3	item4	item5	item6
-VisitsA	Laboratory	1				10	15
-VisitsC	Laboratory	1		120
-VisitsD	Laboratory	1				12	16
-VisitsA	Medication	1	a	10	2011-01-01	30	20
-VisitsA	Medication	1	b	5	2011-01-01
-VisitsA	Medication	2	c	8	2011-01-01
-VisitsA	Medication	1	a	10	2011-01-01
-VisitsB	Medication	2	c	8	2011-01-01
-VisitsC	Medication	1		110
-VisitsC	Medication							test
\ No newline at end of file
+StudySubjectID	EventName	CRFname	CRFversion	item1	item2	item3	item4	item5	item6
+s1	VisitsA	Laboratory	1				10	15
+s2	VisitsC	Laboratory	1		120
+s3	VisitsD	Laboratory	1				12	16
+s3	VisitsA	Medication	1	a	10	2011-01-01	30	20
+s3	VisitsA	Medication	1	b	5	2011-01-01
+s6	VisitsA	Medication	2	c	8	2011-01-01
+s7	VisitsA	Medication	1	a	10	2011-01-01
+s7	VisitsB	Medication	2	c	8	2011-01-01
+s8	VisitsC	Medication	1		110
+s4	VisitsC	Medication							test
```

Comment:
This red crossed circle with return is a sign that you forgot to leave empty line at the end of your file. You can configure Intellij to do it automatically. 
But since it is a tab delimited file I guess you wanted to avoid handling empty line.